packages/deprecated-component-library-helpers/styles/type.scss

```diff
@@ -286,7 +286,8 @@ $ca-ideal-locale-he-font-family: "Ideal Sans A", "Ideal Sans B", Tahoma,
 ) {
   @include ca-type(
     $size-ratio: $size / 16,
-    $family: $kz-typography-heading-1-font-family,
+    $family:
+      "#{$kz-typography-heading-1-font-family}, Helvetica, Arial, sans-serif",
```

Comment:
I don't know if `Helvetica, Arial, sans-serif` is the latest best practice for fallbacks (I copied it from above in this file, which was probably written several years ago). THere might be more recent system fonts (Avenir?) which are closer to Greycliff,  but it's much better than nothing and should be able to test our theory that the lack of fallbacks is causing PDF rendering issues.

Reply:
The rest of the PR will be be removed once approved (might need a re-tick)

Reply:
_Update_: gah, this is totally wrong (good thing I QA'ed it!)

It ends up as `font-family: "Greycliff, Helvetica, Arial, sans-serif"`, but we want `font-family: Greycliff, Helvetica, Arial, sans-serif`

I'm not sure how to do that without SCSS thinking `Helvetica, Arial, sans-serif` are extra function arguments 🤔

Reply:
Could you alternate between double and single quotes? Like have one delimiting the string and the others surrounding the font families?

Reply:
Ah, I missed this. I pushed a change thought that fixes it. I think that might have the same problem though, where scss gets confused between whether something is a string or a list of function arguments.

Reply:
You can still use interpolation here, it's just being sensitive about your quotes. 

```
$font: 'Greycliff CF';

.thing { 
  font-family: "#{$font}", Helvetica, Arial, sans-serif;
}
```